scripts/doctor-aggregate-packages.mjs

```diff
@@ -68,6 +68,7 @@ async function patch(pkgJsonPath, pkgJson, packages, dependenciesKey) {
     ...(pkgJson[dependenciesKey] ?? {}),
     ...Object.fromEntries(
       packages
+        .filter((pkg) => pkg.private !== true)
```

Comment:
Should this be based on if the current project is itself private?

Reply:
That could make sense but the script is really just to automate the docs and aggregate functionless package. We can remove this doctor script probably.